src/elements/content-uploader/ItemList.scss

```diff
@@ -5,3 +5,7 @@
     border-bottom: 1px solid $lighter-grey-blue;
     outline: none;
 }
+
+.bcu-item-list-actionremovecell-column {
```

Comment:
Another approach to this to make it more generic (in case we add more actions in the future) would be to name it `bcu-item-list-action-column` and do a CSS selector with :last-child like `.bcu-item-list-action-column:last-child { margin-right: 18px }` to make sure it only applies to the last action column